src/components/Upgrade/UpgradeModal.js

```diff
@@ -33,8 +33,10 @@ const UpgradeModal = React.memo(({ visible, onUpgrade, onClose }) => {
             padding={0}
             width={Math.min(1055, width - 40)}
             visible={visible}
-            overlayColor="rgb(37, 49, 77, 0.75)"
             onClose={onClose}
+            css={`
+              z-index: 1;
```

Comment:
Maybe we can use a higher z-index here to move this modal above the HelpScout beacon; having the arrow button and the HelpScout beacon so close by to each other on mobile isn't very nice :(

<img width="401" alt="Screen Shot 2019-08-28 at 4 52 25 PM" src="https://user-images.githubusercontent.com/4166642/63866873-3ad75e80-c9b4-11e9-95cb-e15bafbf6c78.png">